autotst/species.py

```diff
@@ -433,12 +433,34 @@ def get_angles(self):
         self.angles = angles
         return self.angles
 
-    def get_torsions(self):
+    def get_torsions(self,tol=5.0):
         """
         A method for identifying all of the torsions in a conformer
         """
+
+        def check_angles(tor):
+            """
+            Checks the two angles in the torsion to  
+            """
+            i,j,k,l = tor
+            angles = ((0,self.ase_molecule.get_angle(i, j, k)), (-1,self.ase_molecule.get_angle(j, k, l)))
```

Comment:
Rather than getting the angles again, we may want to grab the angles from the `Angle` objects that are attached to a conformer already. This may require us to call `get_angles` before (or while) we check the angles.

Reply:
`get_angles` returns a list of all angles, so it's a bit difficult to find the angles of interest